composer.json

```diff
@@ -23,19 +23,19 @@
     "require": {
         "php": "^5.4 || ^7.0",
         "friendsofsymfony/http-cache": "^1.4.5",
-        "symfony/framework-bundle": "^2.3||^3.0"
+        "symfony/framework-bundle": "^2.8||^3.4"
```

Comment:
i don't think we should suddenly restrict dependencies of the maintenance version. instead, we can change the versions in the `.travis.yml` file to only install more specific versions, that usually helps with the out of memory problems.

Reply:
Probably safest. Then we can just adjust require-dev, it should be enough to fix it, on top of corresponding travis.yml changes.